in/command.go

```diff
@@ -146,6 +171,20 @@ func (command *Command) downloadFile(bucketName string, remotePath string, versi
 	)
 }
 
+func (command *Command) createInitialFile(destinationDir string, destinationFile string, data []byte) error {
+	localFile, err := os.Create(filepath.Join(destinationDir, destinationFile))
+	if err != nil {
+		return err
+	}
+	defer localFile.Close()
```

Comment:
By deferring the call to `Close()` like this, we're not checking the error. When writing files, the OS will often buffer writes until the file is closed, meaning that any errors arising from the actually write aren't returned until the close call. See https://joeshaw.org/dont-defer-close-on-writable-files/ for a more detailed explanation of this.

Reply:
Thanks Alex, that's very helpful to know.